appeals/api/src/server/endpoints/documents/documents.service.js

```diff
@@ -70,6 +72,12 @@ export const addDocumentsToAppeal = async (upload, appeal) => {
 		documentsToSendToDatabase
 	);
 
+	for (const document of documentsCreated) {
+		if (document?.documentGuid) {
+			await broadcasters.broadcastDocument(document?.documentGuid, EventType.Create);
+		}
```

Comment:
Should probably log an error if they don't have a Guid, shouldn't all have Guids?

Reply:
Like above, this is mainly an issue with the document API definitions. a document will always have a GUID, but I was trying to avoid this PR getting too big. Again, the entire document endpoint will be refactored with James's work.

For details of the work, some details in https://pins-ds.atlassian.net/wiki/spaces/BO/pages/1528627212/Document+upload+process